### ISSUE TITLE
Your trainable VGG 19  finetune issue

### ISSUE THREAD

**Issue by eng100200** [machrisaa/tensorflow-vgg#31]
@machrisaa I have two questions from you in your Vgg19 trainable implementation. 
I am using your code and your pretrianed, but, now if want to finetune  this network for some other problem using triplet loss, then, how could i implement back propogation?  Is that done automatically using your code. 
I have written a code, if you have time i can send you my code to view.

**Comment by machrisaa** [comment]
Hi @eng100200, Tensorflow will help you to calculate the gradients and do the back propagation for you automatically. So don't worry. Just apply your loss function and see how it goes.

It will be great if you can create a repository can put your code on it.

**Comment by eng100200** [comment]
so, i just upload pre-trained weights and apply the loss function, and sent the loss to the gradient computation then the weights will be automatically updated,,,,coool
Yea sure, after some revisions, i would make me code available here.
thanks for quick reply.

**Comment by eng100200** [comment]
@machrisaa thank you,,, it means i  just need to upload pre-trained weights and apply the loss function, and sent the computed loss for gradient computation, which then update the weights automatically,,,,,,,,cool
Yea sure, after some revisions, i would make my code available here.
thanks for quick reply.
